mlrun/track/__init__.py

```diff
@@ -11,8 +11,23 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+# flake8: noqa  - this is until we take care of the F401 violations with respect to __all__ & sphinx
 
-from .tracker import Tracker
 from .base_tracker import BaseTracker
-from .tracker_manager import TrackerManager, trackers_manager
-from .platform_trackers.mlflow_tracker import MLFlowTracker
+from .tracker import Tracker
+from .tracker_manager import TrackerManager
+from .trackers import MLFlowTracker
+
+
```

Comment:
Think of the contributer who will need to add a new tracker. He will need to add an if to the function and you increase the chance of errors. You can simplify it by guiding him what to do for minimal hazard:

```suggestion
# Add a tracker to this list for it to be added into the global trackers manager:
_AVAILABLE_TRACKERS = [MLFlowTracker]
```